### PR TITLE
Add authors taxonomy for blog posts

### PR DESCRIPTION
**Pull request by earnestma**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change. [?]

### Description

Hi and thanks for this wonderful theme! This PR adds an "authors" taxonomy for blog posts. The person/ user icon shows before tags/categories, and can be browsed like any other taxonomy.

Here are some screenshots:
ss1

ss2

ss3

Configuration:

- toml:
```
+++
authors = ["author 1", "author 2"]
+++
```

- yaml:
```
---
authors: ["author 1"]
---
```

#### General

- Added `{{ with .Page.Params.Authors }}{{ partial "taxonomy/authors.html" . }}{{ end }}` to `layouts/posts/single.html`
- Added partial `taxonomy/authors.html` with the fa-user icon
- Added en (English) string for author
- Added authors to default post archetype
- Added the taxonomy to the exampleSite `config.toml` file

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Review by carvalholeo — APPROVED** [review]

**Comment by luizdepra** [comment]
Nice. Could you please add some authors to one or two post inside the `exampleSite` demo? That way we can see this new feature in action.

**Comment by earnestma** [comment]
sure!

File: exampleSite/resources/_gen/assets/scss/scss/coder-dark.scss_9e20ccd2d8034c8e0fd83b11fb6e2bd5.content [hunk 1]
```diff
@@ -0,0 +1,126 @@
+body.colorscheme-dark {
```
Comment (luizdepra) [inline exampleSite/resources/_gen/assets/scss/scss/coder-dark.scss_9e20ccd2d8034c8e0fd83b11fb6e2bd5.content:1]: You've added these scss files by mistake. Please, remove the `resources` folder from inside `exampleSite`.
Also, there is a simple conflict to solve in `CONTRIBUTORS.md`.

**Comment by luizdepra** [comment]
Nicely done! Thanks.

**Pull request merged**